src/main/kotlin/org/malachite/estella/services/SecurityService.kt

```diff
@@ -0,0 +1,77 @@
+package org.malachite.estella.services
+
+import io.jsonwebtoken.Jwts
+import io.jsonwebtoken.SignatureAlgorithm
+import org.malachite.estella.commons.models.people.User
+import org.malachite.estella.people.domain.UserRepository
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.stereotype.Service
+import java.sql.Date
+import javax.servlet.http.Cookie
+import javax.servlet.http.HttpServletResponse
+
+@Service
+class SecurityService(@Autowired val userRepository: UserRepository) {
```

Comment:
Jest jakiś powód, dla którego używasz UserRepository, a nie User Service?

Reply:
Brak obsługi nie istnienia elementu XD.  Wszędzie pobieraliśmy repository w servicach więc przyjąłem że tak się robi XD

Reply:
Znaczy niby tak, ale jak jest już jakiś service, to warto z niego skorzystać. Nie byłoby takiej sytuacji, że coś się w servicie zmieniło, a o security zapomnieliśmy i walnie. Ogólnie jak czegoś brakuje, to możesz dopisać, nikt Cię za to nie zgani XDDD

Reply:
Znaczy no ok ale dla getów to nie widzę problemu że jest to repository, a z tego co zrozumiałem jak gadałem z Tomkiem to przyjęliśmy zasadę że jak service nie znajdzie elementu to rzuca exception, więc dopisywanie metody która będzie inaczej się zachowywała w serwisie niż mamy w konwencji jest nienajlepszym pomysłem XD